### PR TITLE
Environment override default

### PR DESCRIPTION
**Pull request by psantos1113**
Create a stored procedure to set the default for the environment variable override to true

### CODE REVIEW

File: src/main/resources/db/changelog/alert/4.0.0/changelog-configuration-stored-procedures.xml [hunk 1]
```diff
@@ -364,4 +364,26 @@
...
+    <!-- Create job migration procedure -->
+    <changeSet author="psantos" id="1545000000000-4">
+        <createProcedure
+                dbms="h2"
+                relativeToChangelogFile="true"
+                schemaName="ALERT"
+                procedureName="INSERT_FIELD_VALUE"
+        >
+            CREATE ALIAS INSERT_GLOBAL_FIELD_VALUE AS $$
+                java.lang.Void insertGlobalFieldValue(final java.sql.Connection connection, final java.lang.String descriptor_name, final java.lang.String fieldKey, final java.lang.String fieldValue) throws java.sql.SQLException {
```
Comment (gkillough) [inline src/main/resources/db/changelog/alert/4.0.0/changelog-configuration-stored-procedures.xml]: This is minor, but you're using both snake_case and camelCase for the method parameters.

File: src/main/resources/db/changelog/alert/4.0.0/changelog-configuration-stored-procedures.xml [hunk 2]
```diff
@@ -364,4 +364,26 @@
...
+                    if (fieldValue != null &amp;&amp; !fieldValue.isBlank()) {
+                        try (final java.sql.Statement insertIntoFieldValues = connection.createStatement()) {
+                            final java.lang.String insertStatement = java.lang.String.format("INSERT INTO ALERT.FIELD_VALUES (CONFIG_ID, FIELD_ID, FIELD_VALUE) VALUES (GET_ID_FOR_REGISTERED_DESCRIPTOR_NAME('%s'), GET_FIELD_ID_FOR_SOURCE_KEY('%s'), '%s')",
```
Comment (gkillough) [inline src/main/resources/db/changelog/alert/4.0.0/changelog-configuration-stored-procedures.xml]: I think `GET_ID_FOR_REGISTERED_DESCRIPTOR_NAME(...)` returns the ID from the REGISTERED_DESCRIPTORS table, so this won't be the CONFIG_ID.

File: src/main/resources/db/changelog/alert/4.0.0/changelog-configuration-global-migration.xml [hunk 3]
```diff
@@ -252,4 +252,17 @@
         </sql>
     </changeSet>
 
-</databaseChangeLog>
\ No newline at end of file
+    <!-- Global Scheduling Config -->
+    <changeSet author="psantos" id="1548361428926-5">
```
Comment (gkillough) [inline src/main/resources/db/changelog/alert/4.0.0/changelog-configuration-global-migration.xml]: Just to confirm: a config for this descriptor doesn't exist before this, right?

Comment (psantos1113) [reply src/main/resources/db/changelog/alert/4.0.0/changelog-configuration-global-migration.xml]: Correct I tried to put the stored procedure call in the changelog-configuration-initialization.xml file first but no configuration exists for the settings component at the time that xml file is processed.  So I needed to do the same thing that was done with the other default configurations.

File: src/main/resources/db/changelog/alert/4.0.0/changelog-configuration-stored-procedures.xml (hunk 1 above)
Comment (psantos1113) [reply src/main/resources/db/changelog/alert/4.0.0/changelog-configuration-stored-procedures.xml]: I'll fix it

**Pull request merged**